spec/PeriodicBackgroundSync-index.bs

```diff
@@ -93,3 +93,11 @@ The <dfn>tag</dfn> is a DOMString. Within one [=list of periodicsync registratio
 The options are a dictionary containing [=minInterval=], a long long. <dfn>minInterval</dfn> is used to specify the minimum interval, in milliseconds, at which the periodic synchronization should happen.
 
 The <dfn>registration state</dfn> is one of <dfn>pending</dfn>, <dfn>waiting</dfn>, <dfn>firing</dfn>, or <dfn>reregisteredWhileFiring</dfn>. It is initially set to [=pending=].
+
+
+Resource Usage {#resources}
+============================
+A website will most likely download resources from the network when processing a <code>periodicsync</code> event. The underlying operating system MAY launch the user agent to dispatch these events, and MAY keep it awake for a pre-defined duration to allow processing of the events. Both cause battery drain.
+The user agent SHOULD cap the frequency of these events to limit resource usage by websites when the user has navigated away.
```

Comment:
The recommendation should include capping the execution time of the event, as well as perhaps suggesting other technologies (Background Fetch?) for downloading larger amounts of data.

Reply:
done.

Reply:
I'm not able to link to the Background Spec. Please could you take a look at the link-defaults to see what I'm missing? I've tried these as well:
in a pre block:
    spec:background-fetch; type:dfn; text:background fetch;url:https://wicg.github.io/background-fetch/#background-fetch
and
inline:
    {{BackgroundFetchManager}}

Reply:
You're never defining `BackgroundFetchManager` anywhere, so that's expected. I think the syntax has to be something like:

```html
<pre class="anchors">
spec: background-fetch; urlPrefix: https://wicg.github.io/background-fetch/
  type: interface
    text: BackgroundFetchManager; url: #background-fetch-manager
</pre>
```

Based on:
https://tabatkins.github.io/bikeshed/#link-types
https://tabatkins.github.io/bikeshed/#link-types

Reply:
ok I'll do that. I got a comment on a previous PR to not use anchors and was trying to avoid that. I think there's no way around that here.